packages/p2p/src/components/my-ads/form-ads.jsx

```diff
@@ -29,6 +32,13 @@ const FormAds = observer(() => {
         );
     }
 
+    const onSubmit = (values, props) => {
+        my_ads_store.handleSubmit(values, props);
+        waitWS('p2p_advert_create').then(() => {
+            setDialogOpen(true);
+        });
```

Comment:
- Could we handle this in the store, i.e. in `handleSubmit` itself? Then we don't have to have an ugly `waitWS` but can just wait for the promise to resolve. 
- You could create a new observable for whether the dialog is open in the store.
- You can get rid of the extra `is_warning_visible` by just purging the `api_error_message` once the user has closed the dialog, no need to keep that in memory.